src/__experimental__/components/date/__snapshots__/date-picker.spec.js.snap

```diff
@@ -249,6 +249,10 @@ exports[`StyledDayPicker i18n translation renders properly 1`] = `
           "slateTint": [Function],
           "whiteOpacity": [Function],
         },
+        "picklist": Object {
```

Comment:
Theme object will be removed from snapshot when https://github.com/Sage/carbon/pull/3843 is merged